apps/fxc-front/src/app/components/ui/track-modal.ts

```diff
@@ -45,6 +46,14 @@ export class TrackModal extends connect(store)(LitElement) {
               >
                 <i class="las la-user-astronaut la-2x" style=${`color: ${sel.trackColors(this.state)[track.id]}`}></i
                 >${track.name}
+                <span slot="end" @click=${() => this.handleScore(track)}>
```

Comment:
My first idea was to have an "optimization" / "socre" button in the planning menu - because this is where the score will be shown
<img width="163" alt="image" src="https://github.com/vicb/flyxc/assets/248818/e802ea60-a85e-4ef2-949b-970458161e61">

but if you think here is better why not.

I'll let you decide on which icon to use.

Reply:
Score computation is now in the planner menu.